uber/models.py

```diff
@@ -1654,6 +1654,14 @@ class MPointsForCash(MagModel):
     when        = Column(UTCDateTime, default=lambda: datetime.now(UTC))
 
 
+class PromoCode(MagModel):
+    price           = Column(Integer, default=0)
+    uses            = Column(Integer, default=1)
+    code            = Column(UnicodeText)
+    users           = Column(UnicodeText, default="")
```

Comment:
Eli is the DB master so I defer to him

One thing slightly non-obvious is Dan is trying to make it so there are promo codes given out which can be used i.e. 100 times and then are used up.  So 100 different attendees can use the promo code "GOLDENAXE"